sublime/User/SublimeLinter.sublime-settings

```diff
@@ -1,78 +1,25 @@
+// SublimeLinter Settings - User
 {
-    "user": {
-        "debug": false,
-        "delay": 0.25,
-        "error_color": "D02000",
-        "gutter_theme": "none",
-        "gutter_theme_excludes": [],
-        "lint_mode": "background",
-        "linters": {
-            "coffee": {
-                "@disable": false,
-                "args": [],
-                "excludes": []
-            },
-            "haml": {
-                "@disable": false,
-                "args": [],
-                "excludes": []
-            },
-            "json": {
-                "@disable": false,
-                "args": [],
-                "excludes": [],
-                "strict": true
-            },
-            "php": {
-                "@disable": false,
-                "args": [],
-                "excludes": []
-            },
-            "rubocop": {
-                "@disable": false,
-                "args": [],
-                "excludes": []
-            },
-            "scss": {
-                "@disable": false,
-                "args": [],
-                "exclude-linter": "",
-                "excludes": [],
-                "include-linter": ""
-            },
-            "xmllint": {
-                "@disable": false,
-                "args": [],
-                "excludes": []
-            }
-        },
-        "mark_style": "outline",
-        "no_column_highlights_line": false,
-        "passive_warnings": false,
-        "paths": {
-            "linux": [],
-            "osx": [
-                "/Users/fuzzy/.rbenv/shims"
-            ],
-            "windows": []
-        },
-        "python_paths": {
-            "linux": [],
-            "osx": [],
-            "windows": []
-        },
-        "rc_search_limit": 3,
-        "shell_timeout": 10,
-        "show_errors_on_save": true,
-        "show_marks_in_minimap": true,
-        "syntax_map": {
-            "html (django)": "html",
-            "html (rails)": "html",
-            "html 5": "html",
-            "php": "html",
-            "python django": "python"
-        },
-        "warning_color": "DDB700",
-        "wrap_find": true
+  "paths": {
+      "linux": [],
+      "osx": [
+          "~/.rbenv/shims"
+      ],
+      "windows": [],
+  },
+  "linters": {
+    "rubocop": {
+      // "use_bundle_exec": true,
+      "excludes": "*/*.slim"
+    },
+    "slimlint": {
+      "use_bundle_exec": true,
+      "env": { "SLIM_LINT_RUBOCOP_CONF": ".rubocop.yml"}
+    },
+
+    "eslint": {
+      "disable_if_not_dependency": true,
+      "env": { "PATH": "/usr/local/opt/node@8/bin" }
```

Comment:
This one sounds awkward. Node8 is really old and I'd expect `nvm` to be used into projects. I'm i missing something?